.github/workflows/ci-pr-title.yaml

```
@@ -0,0 +1,94 @@
+name: CI Check Title
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  title-lint:
+    name: Validate PR title
+    runs-on: [default]
+    steps:
+      - name: CI Check Title
+        uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          wip: true
+          # Configure which types are allowed (newline-delimited).
+          # Default: https://github.com/commitizen/conventional-commit-types
+          types: |
+            build
+            chore
+            fix
+            feat
+            merge
+            publish
+            release
+            refactor
+            research
+            style
+            test
+          # Configure which scopes are allowed (newline-delimited).
+          # These are regex patterns auto-wrapped in `^ $`.
+          scopes: |
+            clusterkubeconfigs
+            clusters
+            organizations
+            plugindefinitions
+            pluginpresets
+            plugins
+            teammemberships
+            teamrolebindings
+            teamroles
+            teams
```

I see that @uwe-mayer is using `service-proxy` in some PR
```suggestion
            teams
            service-proxy
```